openprovider/modules/ssl.py

```diff
@@ -4,18 +4,32 @@
 from openprovider.models import SSLProduct, SSLOrder
 
 
+def _domain_validation_methods(methods):
+    if not methods:
+        return None
+
+    items = [E.item(E.hostName(hostname), E.method(method)) for hostname, method in methods.items()]
+    return E.array(*items)
+
+
+def _simple_array(hostnames):
+    items = [E.item(name) for name in hostnames]
+    return E.array(*items)
+
+
 class SSLModule(common.Module):
     """Bindings to API methods in the SSL module."""
 
-    def search_product(self, limit=100, offset=0, **kw):
+    def search_product(self, limit=100, offset=0, with_price=0, with_supported_software=0,
+                       with_description=0):
```

Comment:
Als je niet bekend bent met de API heb je niet echt een mentaal model van **alles**, dus dat is dan wat lastig te beoordelen :).